ITR/data/excel.py

```diff
@@ -269,12 +277,20 @@ def _get_projection(self, company_ids: List[str], projections: pd.DataFrame, pro
 
         return projected_emissions_s1s2
 
+    def _convert_target_data(self, target_data: pd.DataFrame) -> List[ITargetData]:
+        """
```

Comment:
Convert target data to what?
My I suggest, to add a comment or the method, which would explain, what the method does?

Reply:
It's really about taking a  row of "target data" (meaning data about published company targets for decarbonization, i.e. achieving a specific reduction by a given data) and converting it into the ITargetData model.